Source/WTF/wtf/PlatformHave.h

```diff
@@ -1165,6 +1165,7 @@
 
 #if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 130000)
 #define HAVE_SCREEN_CAPTURE_KIT 1
+#define HAVE_NSSHARINGSERVICEPICKER 1
```

Comment:
`NSSharingServicePicker` has been available since macOS 10.8+, so this name is too generic.

I suggest `HAVE_SHARING_SERVICE_PICKER_STANDARD_SHARE_MENU_ITEM` and moving this next to `HAVE_SHARING_SERVICE_PICKER_POPOVER_SPI`.